src/command/giday.command.ts

```
@@ -0,0 +1,287 @@
+import { ChannelMessage } from 'mezon-sdk';
+import { Command } from '@app/decorators/command.decorator';
+import { CommandMessage } from '@app/command/common/command.abstract';
+import { GiDayService, GiDayRandomResult } from '@app/services/giday.service';
+import { GIDAY_MESSAGES } from '@app/messages/giday.messages';
+import { formatMessage, joinMessages, createNumberedList } from '@app/utils/message-formatter.utils';
```

ESLint

[error] Replace `·formatMessage,·joinMessages,·createNumberedList·` with `⏎··formatMessage,⏎··joinMessages,⏎··createNumberedList,⏎`

(prettier/prettier)